From ea940101835bb49ad7a440ef0a4aa69d7bb76f61 Mon Sep 17 00:00:00 2001
From: Xianzhong <b07117@freescale.com>
Date: Tue, 14 Aug 2012 22:29:42 +0800
Subject: [PATCH 041/281] ENGR00219602 Config VPU allocator with GPU video
 pool

USE_GPU_ALLOCATOR indicate VPU request memory with GPU allocator,

Unify the reserved memory for GPU & VPU to ultize memory with max efficiency.

Signed-off-by: Xianzhong <b07117@freescale.com>
---
 imx6/arm2/BoardConfigArm2.mk           | 3 ++-
 imx6/sabresd/SabreSDBoardConfigComm.mk | 3 ++-
 2 files changed, 4 insertions(+), 2 deletions(-)

diff --git a/imx6/arm2/BoardConfigArm2.mk b/imx6/arm2/BoardConfigArm2.mk
index f5b7886..bdb4ec1 100644
--- a/imx6/arm2/BoardConfigArm2.mk
+++ b/imx6/arm2/BoardConfigArm2.mk
@@ -50,4 +50,5 @@ TARGET_USERIMAGES_USE_EXT4 := true
 # atheros 3k BT
 BOARD_USE_AR3K_BLUETOOTH := false
 
-USE_ION_ALLOCATOR := true
+USE_ION_ALLOCATOR := false
+USE_GPU_ALLOCATOR := true
diff --git a/imx6/sabresd/SabreSDBoardConfigComm.mk b/imx6/sabresd/SabreSDBoardConfigComm.mk
index 7f86929..55a7769 100755
--- a/imx6/sabresd/SabreSDBoardConfigComm.mk
+++ b/imx6/sabresd/SabreSDBoardConfigComm.mk
@@ -68,4 +68,5 @@ endif
 # atheros 3k BT
 BOARD_USE_AR3K_BLUETOOTH := true
 
-USE_ION_ALLOCATOR := true
+USE_ION_ALLOCATOR := false
+USE_GPU_ALLOCATOR := true
-- 
1.8.0

